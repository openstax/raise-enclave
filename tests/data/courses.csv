id,name,research_participation
2,Algebra 1 (Research Dev),0
3,Algebra 2 (Research Dev 2),1