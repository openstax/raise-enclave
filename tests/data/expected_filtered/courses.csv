id,name
3,Algebra 2 (Research Dev 2)
