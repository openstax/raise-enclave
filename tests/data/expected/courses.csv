id,name
2,Algebra 1 (Research Dev)
3,Algebra 2 (Research Dev 2)